src/plays/personal-profile-card/PersonalProfileCard.tsx

```diff
@@ -0,0 +1,66 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import React, { useState, ChangeEvent, useEffect } from 'react'
+
+import * as Storage from "./local-storage"
+import ProfileType from './types'
+import ProfileCard from './components/profile-card'
+import ProfileForm from './components/profile-form'
+import './index.css'
+
+const PersonalProfileCard = (props: any) => {
+  const [profileCard, setProfileCard] = useState<ProfileType>(null)
+  const [value, setValue] = useState(null)
+  const [key, setKey] = useState<number>(0)
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement> | ChangeEvent<HTMLTextAreaElement>):void => {
+    setValue((value: Object) => {
+      return { ...value, [e.target.name]: e.target.value }
+    })
+  }
+
+  useEffect(() => {
+    setProfileCard(Storage.get('profile_card'))
+    Storage.get('profile_card') !== null && setValue(Storage.get('profile_card'))
+  }, [key])
```

Comment:
what is the role for `key`? is it to make a force re-render?

Reply:
`key` is used for retrieve data from the localstorage every time the form updates.